### PR TITLE
Add week 4 part-time schedule

### PR DESCRIPTION
**Pull request by dunxen**
I tried to split up some of the reading. Not sure if it's still too dense or now too light. Also, note the TODOs.

### CODE REVIEW

**Comment by tvpeter** [comment]
Hi @dunxen, I have drafted the below goals and prerequisites. Could you check if they are good? I'm sorry I don't know how to add them to the PR.

```---
description: Continuation of Mastering Lightning
---

# Week 4

This week, we'll be building on the previous knowledge of Lightning Network by continuing with "Mastering Lightning" Book. Discussions will be in small groups as usual as you answer discussion questions. 

A web-reader-friendly hosted version of the book can be found at [lnbook.256k1.dev](https://lnbook.256k1.dev), which can be accessed for educational purposes using username and password `qala:lightning`.

## Prerequisites

* All the previous weeks' developer environment for both Bitcoin and Lightning Networks.

## Goals

At the end of the week, we hope that you will:

* Understand the architecture of Lightning Network
* Understand Payment Channels
  * Constructing a Payment Channel
  * Sending Payment accross the Channel
  * The Commitment Transaction
  * Advancing the Channel State
  * Closing the Channel
* Understand Routing on a network of Payment Channels
  * Routing a Payment
  * Routing vs pathfinding
  * Creating a network of Payment Channels
  * Fairness Protocol
  * Hash Time-Locked Contracts (HTLCs)
```

**Comment by dunxen** [comment]
> Hi @dunxen, I have drafted the below goals and prerequisites. Could you check if they are good? I'm sorry I don't know how to add them to the PR.

Thanks Peter! Usually you can just open a PR against my branch on my fork and I could upstream it from there. But it's no hassle. Alternatively you can make a suggestion review comment and commit and I can commit it from the GitHub UI.

File: part-time-schedule/week-4.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+description: More Mastering Lightning
+---
+
+# Week 4
+
```
Comment (tvpeter) [inline part-time-schedule/week-4.md:6]:
```suggestion
This week, we'll be building on the previous knowledge of Lightning Network by continuing with "Mastering Lightning" Book. Discussions will be in small groups as usual as you answer discussion questions. 

A web-reader-friendly hosted version of the book can be found at [lnbook.256k1.dev](https://lnbook.256k1.dev), which can be accessed for educational purposes using username and password `qala:lightning`.
```

File: part-time-schedule/week-4.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+## Prerequisites
+
+(TODO - We already mentioned polar for the previous week which would be needed for the workshop below. Do we need anything else?)
```
Comment (tvpeter) [inline part-time-schedule/week-4.md:9]:
```suggestion
* All the previous weeks' developer environments for both Bitcoin and Lightning Networks.
```

File: part-time-schedule/week-4.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Goals
+
+TODO
```
Comment (tvpeter) [inline part-time-schedule/week-4.md:13]:
```suggestion
At the end of the week, we hope that you will:

* Understand the architecture of Lightning Network
* Understand Payment Channels
  * Constructing a Payment Channel
  * Sending Payment accross the Channel
  * The Commitment Transaction
  * Advancing the Channel State
  * Closing the Channel
* Understand Routing on a network of Payment Channels
  * Routing a Payment
  * Routing vs pathfinding
  * Creating a network of Payment Channels
  * Fairness Protocol
  * Hash Time-Locked Contracts (HTLCs)
```

**Review by tvpeter — COMMENTED** [review]
Added suggestive comments. Hopefully, the formatting is still in order. Thank you.

**Pull request merged**